src/hypothesis/strategies.py

```diff
@@ -945,11 +955,26 @@ def builds(target, *args, **kwargs):
     the target.
 
     """
+    if target_and_args:
+        target, args = target_and_args[0], target_and_args[1:]
+        if not callable(target):
+            raise InvalidArgument('The first positional argument to builds() must be a callable'
```

Comment:
Needs a space at the end of this string, to avoid joining words when it's concatenated.